pom.xml

```diff
@@ -372,6 +372,21 @@
                       <property name="severity" value="error"/>
                       <property name="eachLine" value="true"/>
                     </module>
+                    <module name="TreeWalker">
+                      <!-- Import settings: getting rid of redundant import, forbid star notation, defined order of imports. -->
+                      <module name="RedundantImport"/>
+                      <module name="AvoidStarImport">
+                        <property name="excludes" value="org.junit.Assert,org.assertj.core.api.Assertions,org.mockito.Mockito"/>
+                        <property name="allowClassImports" value="false"/>
+                        <property name="allowStaticMemberImports" value="false"/>
+                      </module>
+                      <module name="CustomImportOrder">
+                        <property name="customImportOrderRules"
+                                  value="STANDARD_JAVA_PACKAGE###THIRD_PARTY_PACKAGE###STATIC"/>
+                        <property name="sortImportsInGroupAlphabetically" value="true"/>
+                        <property name="separateLineBetweenGroups" value="true"/>
+                      </module>
```

Comment:
Hi @ge0ffrey ,
and yes, and no. Both of them have restrictions for Redundant Imports, proposed by Ondrej. Also Ondrej's config contains old java packages restrictions (like java.io and so on), but this PR not. On the other hand this rules contains Avoid Star Import module (with exceptions according to actual IDEA settings) and strict Imports Order (1) in addition.
(1.)
> **import** *java.*;
> **import** *javax.*; 
>
> **import** *other.packages*;
>
> **import static** *all.statics*;

So if you are using current IDEA settings it haven't to make any troubles to you. (Except this https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/pull/342/files#diff-9d663bfb476dd8a86f4f213da123bfa5R4 fix)

Any way if your repository is affected you can use fixed IDEA settings for automated imports organizing:
Update your IDEA configuration and use Ctrl+Alt+O for single file or left mouse button on project name in Project Explorer -> Optimize Imports) for whole project, also you can set Optimize import on the fly option (Settings -> Editor -> General -> Auto Import -> Optimize import on the fly) and don't care how imports organized for new files :)